contrib/buildsystems/CMakeLists.txt

```diff
@@ -178,10 +192,19 @@ endif()
 
 find_program(MSGFMT_EXE msgfmt)
 if(NOT MSGFMT_EXE)
-	set(MSGFMT_EXE ${CMAKE_SOURCE_DIR}/compat/vcbuild/vcpkg/downloads/tools/msys2/msys64/usr/bin/msgfmt.exe)
-	if(NOT EXISTS ${MSGFMT_EXE})
-		message(WARNING "Text Translations won't be built")
-		unset(MSGFMT_EXE)
+	if(NO_GETEXT)
+		if(MSGFMT_EXE)
+			message(WARNING "MSGFMT_EXE is ignored under NO_GETTEXT")
+			unset(MSGFMT_EXE)
+		endif()
+	else()
+		if(USING_VCPKG)
+			set(MSGFMT_EXE ${CMAKE_SOURCE_DIR}/compat/vcbuild/vcpkg/downloads/tools/msys2/msys64/usr/bin/msgfmt.exe)
+		endif()
+		if(NOT EXISTS ${MSGFMT_EXE})
+			message(WARNING "Text Translations won't be built")
+			unset(MSGFMT_EXE)
+		endif()
```

Comment:
There is a missing `endif()` here, I think, for the `else()` above.

And I'd like to avoid `find_program(MSGFMT_EXE)` altogether if `NO_GETTEXT` is set...

Reply:
I made some updates with your comments in mind, I think the version I have is a bit more what we want, I changed the text output from a warning to a status because this should be normal operation if we're operating under NO_GETTEXT and I felt issuing a warning for that would be a bit weird.